storage/src/nodestore/mod.rs

```diff
@@ -779,7 +783,7 @@ mod tests {
         value: Some(vec![9, 10, 11].into_boxed_slice()),
         children: from_fn(|i| {
             if i == 15 {
-                Some(Child::AddressWithHash(nonzero!(1u64), std::array::from_fn::<u8, 32, _>(|i| i as u8).into()))
+                Some(Child::AddressWithHash(LinearAddress::new(1).unwrap(), std::array::from_fn::<u8, 32, _>(|i| i as u8).into()))
```

Comment:
The nonzero! macro was really handy here, as it keeps the code focused on what we're doing. You can probably create a macro here for this, something like this:

```
macro_rules! linear_address {
    ($addr:expr) => {
        LinearAddress::new(nonzero!($addr))
    };
}
```
Then you end up with this much cleaner code at the callsite:

```suggestion
                Some(Child::AddressWithHash(linear_address!(1), std::array::from_fn::<u8, 32, _>(|i| i as u8).into()))
```

It might be a little trickier; you'll have to dig into the nonzero! macro to see exactly what types it expects.